src/db/mongo.ts

```
@@ -49,8 +49,12 @@ export class MongoHelper {
 
 export async function test() {
   const chatDb = new MongoHelper()
-  const newChat: ChatHistory = {user_id: 1, role: 'user', message: 'hello', created_at: new Date()}
+  const newChat: ChatHistory = {user_id: 1, role: 'user', model: 'gpt4.0', message: 'hello', created_at: new Date()}
   const userId = await chatDb.insertOne(newChat)
   console.log('Inserted User ID:', userId)
+  const history = await chatDb.findAllSorted(String(1060046125828341760))
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 1060046125828341800

(lint/correctness/noPrecisionLoss)